executor_manager/executors/docker/git_cache_volume_manager.py

```
@@ -0,0 +1,638 @@
+def get_volume_metadata(volume_name: str) -> Optional[Dict[str, str]]:
+    """
+    Get metadata labels from a volume.
+
+    Args:
+        volume_name: Name of the volume
+
+    Returns:
+        Dictionary of labels or None if volume doesn't exist
+    """
+    try:
+        result = subprocess.run(
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)